requirements/base.txt

```diff
@@ -0,0 +1,8 @@
+unipath>=1.1,<=2.0
+django>=1.8,<=1.9
+git+https://github.com/cfpb/django-sheerlike.git#egg=sheerlike
+six==1.9.0
+python-dateutil==2.4.2
```

Comment:
@dpford @kurtw @rosskarchner Hey so this version conflicts with sheer's version of 2.1  are there any updates to this package that are being used that aren't in 2.1 ?